sqlmesh/dbt/test.py

```diff
@@ -121,27 +121,32 @@ def to_sqlmesh(self, context: DbtContext) -> AuditType:
         skip = not self.enabled
         blocking = self.severity == Severity.ERROR
 
-        audit = Audit(
-            name=self.name,
-            dialect=context.dialect,
-            skip=skip,
-            blocking=blocking,
-            query=query,
-            jinja_macros=jinja_macros,
-        )
-        audit._path = self.path
-
-        if self.owner:
-            return audit
+        audit: Audit
+        if self.is_standalone:
+            audit = StandaloneAudit(
+                name=self.name,
+                dialect=context.dialect,
+                skip=skip,
+                query=query,
+                jinja_macros=jinja_macros,
+                depends_on={model.sql_name for model in test_context.refs.values()}.union(
+                    {source.sql_name for source in test_context.sources.values()}
+                ),
+                tags=self.tags,
+                # TODO node values
```

Comment:
what's node values?

Reply:
Node class fields. I plan to add those in the follow-on PR.